src/index.html

```diff
@@ -1,12 +1,45 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--30 calendar--start-day--wed">
+      <div class= "calendar__cell"> </div>
+      <div class= "calendar__cell"> </div>
+      <div class= "calendar__cell"> </div>
+      <div class= "calendar__cell"> </div>
+      <div class= "calendar__cell"> </div>
+      <div class= "calendar__cell"> </div>
+      <div class= "calendar__cell"> </div>
+      <div class= "calendar__cell"> </div>
+      <div class= "calendar__cell"> </div>
+      <div class= "calendar__cell"> </div>
```

Comment:
Remove the extra space between the class attribute and its value for better code readability.